src/tox_uv/_venv.py

```diff
@@ -113,7 +113,13 @@ def _default_pass_env(self) -> list[str]:
 
     def create_python_env(self) -> None:
         base = self.base_python.version_info
-        version_spec = f"{base.major}.{base.minor}" if base.minor else f"{base.major}"
+        version_spec = (
+            sys.executable
```

Comment:
I don't like this because what if the executable does not match the requested version... The approach you are proposing here is essentially side stepping any version checking.

Reply:
I'm sorry, I don't really understand this comment. Does the `if (base.major, base.minor) == sys.version_info[:2]` check not do exactly what you are asking - make sure the version of `sys.executable` matches the requested version?

Reply:
Never mind this comment.